Lib/tkinter/font.py

```diff
@@ -184,7 +184,7 @@ def metrics(self, *options, **kw):
 def families(root=None, displayof=None):
     "Get font families (as a tuple)"
     if not root:
-        root = tkinter._default_root
+        root = tkinter._get_default_root('use font.families()')
```

Comment:
```suggestion
        tk = tkinter._get_default_root('use font.families()').tk
```
Similar to what you did in `__init__.py` (which I thought was a good idea)

Reply:
This code differs from the code in `__init__.py`, because we have the root parameter. So we would need to add a branch:
```
else:
    tk = root.tk
```
It is larger change of code and I am not sure it is worth.